overlord/snapstate/backend/mountunit_test.go

```diff
@@ -67,6 +70,21 @@ func (s *FakeSystemd) RemoveMountUnitFile(mountDir string) error {
 	return s.RemoveMountUnitFileResult
 }
 
+type ParamsForListMountUnits struct {
+	snapName, origin string
+}
+
+type ResultForListMountUnits struct {
+	mountPoints []string
+	err         error
+}
+
+func (s *FakeSystemd) ListMountUnits(snapName, origin string) ([]string, error) {
+	s.ListMountUnitsCalls = append(s.ListMountUnitsCalls,
```

Comment:
(nitpick^2) I think it's fine to have this and the following in a single line, go has no hard rules here and we are not really consistent but I personally like (slightly) longer lines over this wrapping. But it's fine, no need to change it, just couldn't not comment on it :)